config-overrides.js

```diff
@@ -5,6 +5,12 @@ const CompressionPlugin = require('compression-webpack-plugin')
 module.exports = function override(config, env) {
   config = rewireStyledComponents(config, env)
 
+  config.module.rules.push({
+    test: /\.mjs$/,
+    include: /node_modules/,
+    type: 'javascript/auto',
+  })
```

Comment:
what is the purpose of this?

Reply:
Webpack didnt compile `.mjs` file. In our imported solana libs, there is lib come with .mjs file instead of `.js` like others. In fact `.mjs` file not different from `.js` file but support `import ... from` natively. So don't to be much concern about it.